src/Workspaces/Core/Portable/Storage/SQLite/Interop/SqlConnection.cs

```diff
@@ -50,10 +50,20 @@ internal class SqlConnection
         /// </summary>
         public bool IsInTransaction { get; private set; }
 
+        /// <summary>
+        /// Whether or not our constructor made it all the way to the end, and thus
+        /// 
+        /// </summary>
+        private readonly bool _constructedCompletely;
```

Comment:
💭 I'm torn on this. Now that allocations have been removed from the constructor, it's no longer possible for this to fail. If others feel it helps, then we can keep it. It's just a NOP in practice.

Reply:
Is that true?  how does thsi work with async-exceptions?

Reply:
Ideally, it doesn't.

Reply:
ok, so we can remove this and just ignore the async-exception case.  presumably it's rare enough to never matter.